src/app/backend/usergroup-management.ts

```diff
@@ -241,59 +241,6 @@ class UserGroupManagementService {
     }
   }
 
-  /**
-   * Saves the user group membership for a specific user group.
-   * @param groupId - The unique identifier of the user group.
-   * @param selectedUsers - The unique identifiers of the users to add to the group.
-   * @returns A list of user group memberships for the specified group.
-   */
-  @transaction
-  @adminRequired
-  static async saveUserGroupMembership(
```

Comment:
this function wasn't getting used anywhere (since I think we're just using the add / remove functions directly) so I went ahead and deleted it